_includes/accordion.html

```diff
@@ -1,5 +1,5 @@
 <div class="usa-accordion {% if include.bordered %}usa-accordion--bordered{% endif %}">
-    <h4 class="usa-accordion__heading">
+    <h3 class="usa-accordion__heading">
```

Comment:
Can we use something like `{: .font-sans-lg }` if the heading level is causing errors?

Switching this from h4 to h3 means that it automatically appears on in-page navigation menu, which we want to avoid. Our content strategy for accordions is that they only are used to address edge cases and condense info that would be distracting to the majority of users. Screenshot:

<img width="847" alt="Screenshot 2024-12-31 at 9 31 22 AM" src="https://github.com/user-attachments/assets/c81f0416-7286-46a5-8f01-32f81b6eb1dd" />